lib/superset/guest_token.rb

```diff
@@ -20,7 +21,7 @@ def params
             "id": embedded_dashboard_id.to_s,
             "type": "dashboard" }
         ],
-        "rls": [],
+        "rls": rls_clause || [], # Ex: [{ "clause": "publisher = 'Nintendo'" }]
```

Comment:
Is the `||` needed here?

Reply:
I added this to handle the case when the class is initiailized with nil value for rls_clause. I also thought about adding a validation for this. Do you suggest adding validation to avoid nil instead of soft handling with || ?

Reply:
That was the reason I suggested to test the nil scenario. If `nil` will not be converted to `[]` then we can keep that.

Reply:
@msxavi as per this - https://github.com/apache/superset/tree/master/superset-embedded-sdk#creating-a-guest-token, seems like the rls clause expects the value to be an array. Hence I added `|| []` to be on safer side